health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/UserAccess.js

```
@@ -134,18 +135,35 @@ function UserAccess({ category, setData, nationalRoles }) {
       selector: (row) => row.email || t("NA"),
-      sortable: true,
+      sortable: false,
     },
     {
       name: t("CONTACT_NUMBER"),
       selector: (row) => {
         return row.number || t("NA");
       },
       sortable: true,
+      sortFunction: (rowA, rowB) => {
+        const numberA = parseInt(rowA.number, 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)